### DISCUSSION TITLE
Use locally encrypted environment for deploy

### DISCUSSION THREAD

**Discussion by ps-20x** [serversideup/spin#128] [Ideas]
The idea would be to have something like `.env.[environment].encrypted` in the codebase. The encryption password can be stored locally in `.vault-password` and in the CI. So the CI can take care of decrypting the `.env`-file and we don't need to work with the `ENV_FILE_BASE64`. So changes will only need to take place in the code and no environment variables need to be adapted in the CI.

**Comment by jaydrogers** [comment]
I have some major changes coming to this structure that I've been working on for the last month. Stay tuned once I have some demos ready https://x.com/jaydrogers

**Reply by ps-20x** [reply]
Perfect thanks. Appreciate your work a lot. Thank you very much
